package.json

```diff
@@ -87,7 +88,7 @@
     ],
     "*.{scss,css}": [
       "yarn format:staged",
-      "yarn lint:styles",
+      "yarn lint:styles:staged",
```

Comment:
```suggestion
      "stylelint --config ./packages/stylelint-config-elements",
```

Maybe this since we won't really need to run the staged command as an npm script?

Reply:
I see, updated.